### PR TITLE
Add cli parsing

### PR DESCRIPTION
**Pull request by hwine**
I added my "standard" python script wrapper. Once we've agreed on the format, I'll do the same for some of the other scripts.
I have 3 orgs I admin, not having to edit the script is a huge win for me. :smile:

### CODE REVIEW

File: admins.py [hunk 1]
```diff
@@ -1,25 +1,61 @@
...
+            print a['login']
+    else:
+        print 'Congrats! All admins have 2FA enabled in %s!' % org
+
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(description=__doc__)
+    parser.add_argument("orgs", nargs='*', default=['mozilla',],
+            help='github organizations to check (defaults to mozilla)')
```
Comment (fwenzel) [inline admins.py]: indentation

File: admins.py [hunk 2]
```diff
@@ -1,25 +1,61 @@
...
 
     admins_with_1fa_url = '%s?%s' % (
-        'https://api.github.com/orgs/%s/members' % 'mozilla',
+        'https://api.github.com/orgs/%s/members' % org,
         urllib.urlencode(params)
     )
     resp = requests.get(admins_with_1fa_url, headers=headers)
+    if resp.status_code not in [200,]:
+        logging.warn("Failed request for %s (code %d)", org,
+                resp.status_code)
```
Comment (fwenzel) [inline admins.py]: indent

File: admins.py [hunk 3]
```diff
@@ -1,25 +1,61 @@
...
     params = {'filter': '2fa_disabled',
               'role': 'admin'}
 
     admins_with_1fa_url = '%s?%s' % (
-        'https://api.github.com/orgs/%s/members' % 'mozilla',
+        'https://api.github.com/orgs/%s/members' % org,
         urllib.urlencode(params)
     )
     resp = requests.get(admins_with_1fa_url, headers=headers)
+    if resp.status_code not in [200,]:
```
Comment (fwenzel) [inline admins.py]: nit, this is a list for no good reason, use (immutable) tuples unless you plan on changing the sequence ;)

**Comment by fwenzel** [comment]
Hey this is great, r+wc (also let me know if I annoy you nit-picking over your Python ;))

**Comment by fwenzel** [comment]
I'll just land that and will iron out the things I just mentioned

**Comment by hwine** [comment]
> let me know if I annoy you nit-picking over your Python
No problem. My python can use some tightening. It's an incentive to reinstall pythonmode which frustrated me one too many times once. :)

**Pull request merged**